src/EnergyPlus/OutputReportTabular.cc

```diff
@@ -14180,6 +14356,41 @@ Label900: ;
 		return StringOut;
 	}
 
+	bool 
+	isNumber(std::string s)
+	{
+		bool is = true;
+		try {
+			std::stof(s);
+		} catch ( std::exception & e ) {
```

Comment:
Great!  Thanks for the insight, that helps, and makes sense.  If @JasonGlazer can move forward with your advised function, that's awesome.

@JasonGlazer I'm not sure about the trailing whitespace issue.  Is there a different way to parse the lines that ensures you don't end up with whitespace padded strings?  Or would you have to trim the strings before passing them in?